recipes/lume-genesis/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "lume-genesis" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 2adfa80ee897903aa1216d2cdf8a831282eb64ce0c3758f43e2e2c9297f5e729
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - bokeh
+    - matplotlib
```

Comment:
Is it OK for this to be `matplotlib-base` instead? Or do you really need the Qt backend

Reply:
Yes, it can be matplotlib-base.

Reply:
@scopatz I have corrected it.